src/classes/RD2_ElevateIntegrationService_TEST.cls

```diff
@@ -580,6 +580,20 @@ public class RD2_ElevateIntegrationService_TEST {
         return new RD2_ElevateIntegrationService();
     }
 
+    /****
+    * @description Sets Elevate Integration service so the integration is enabled
+    * @return RD2_ElevateIntegrationService
+    */
+    public static void configureElevateUserServiceWithPermission() {
```

Comment:
- Let's remove `@return` since the method does not have an output
- Let's rename the method to `enableElevateUserPermissions()`